crates/api/src/post/mark_read.rs

```diff
@@ -2,16 +2,15 @@ use actix_web::web::{Data, Json};
 use lemmy_api_common::{context::LemmyContext, post::MarkPostAsRead, SuccessResponse};
 use lemmy_db_schema::source::post::PostRead;
 use lemmy_db_views::structs::LocalUserView;
-use lemmy_utils::error::{LemmyErrorExt, LemmyErrorType, LemmyResult, MAX_API_PARAM_ELEMENTS};
-use std::collections::HashSet;
+use lemmy_utils::error::{LemmyErrorType, LemmyResult, MAX_API_PARAM_ELEMENTS};
 
 #[tracing::instrument(skip(context))]
 pub async fn mark_post_as_read(
   data: Json<MarkPostAsRead>,
   context: Data<LemmyContext>,
   local_user_view: LocalUserView,
 ) -> LemmyResult<Json<SuccessResponse>> {
-  let post_ids = HashSet::from_iter(data.post_ids.clone());
+  let post_ids = &data.post_ids;
 
```

Comment:
I got rid of all these fairly pointless HashSets (a way to enforce uniqueness of post ids to be marked as read). 

Since we're no longer allowing vectors of marked read post ids via the API, and only through code, these seemed redundant. Also postgres can handle in queries with the same ID with no problem anyway.

Reply:
It looks like this function is unused now.

Reply:
I've removed that now in other PRs.